service/src/main/java/com/commercetools/pspadapter/payone/config/ServiceConfig.java

```diff
@@ -11,76 +16,44 @@
  */
 public class ServiceConfig {
 
-    private final String ctProjectKey;
-    private final String ctClientId;
-    private final String ctClientSecret;
-    private final boolean startFromScratch;
+
+    private final List<String> tenants;
+
     private final String scheduledJobCronShortTimeFrame;
     private final String scheduledJobCronLongTimeFrame;
-    private final String secureKey;
-    private final boolean updateOrderPaymentState;
 
     /**
      * Initializes the configuration.
      *
      * @param propertyProvider to get the parameters from
-     *
      * @throws IllegalStateException if a mandatory parameter is undefined or empty
      */
     public ServiceConfig(final PropertyProvider propertyProvider) {
-        ctProjectKey = propertyProvider.getMandatoryNonEmptyProperty(PropertyProvider.CT_PROJECT_KEY);
-        ctClientId  = propertyProvider.getMandatoryNonEmptyProperty(PropertyProvider.CT_CLIENT_ID);
-        ctClientSecret = propertyProvider.getMandatoryNonEmptyProperty(PropertyProvider.CT_CLIENT_SECRET);
-        scheduledJobCronShortTimeFrame =
+
+        this.tenants = getMandatoryTenantNames(propertyProvider);
+
+        this.scheduledJobCronShortTimeFrame =
                 propertyProvider.getProperty(PropertyProvider.SHORT_TIME_FRAME_SCHEDULED_JOB_CRON)
                         .map(String::valueOf)
                         .orElse("0/30 * * * * ? *");
-        scheduledJobCronLongTimeFrame =
+        this.scheduledJobCronLongTimeFrame =
                 propertyProvider.getProperty(PropertyProvider.LONG_TIME_FRAME_SCHEDULED_JOB_CRON)
                         .map(String::valueOf)
                         .orElse("5 0 0/1 * * ? *");
```

Comment:
because it was designed by the application  requirements to have some fallback cron job schedule, if not specified by the customer.